src/vs/vscode.proposed.d.ts

```diff
@@ -33,7 +33,33 @@ declare module 'vscode' {
 		extensionHostEnv?: { [key: string]: string | null };
 	}
 
-	export type ResolverResult = ResolvedAuthority & ResolvedOptions;
+	export interface Tunnel extends TunnelDescriptor, Disposable {
+		localAddress: Uri;
```

Comment:
Are these really URIs? Can they have a scheme, path, query params, etc? I think a string would do.

Reply:
A string will do, but a lot of our API takes URIs so I thought that they may often get converted into a URI anyway.

Reply:
It will be like if you want to create an http url based off this info, but I don't think it makes sense for this interface to declare that the address is an http-URI, it could also be accessed in other ways.

Reply:
Changed!